app/src/views/FourHundredFour/styles.module.css

```diff
@@ -2,10 +2,12 @@
     display: flex;
     flex-direction: column;
 
+
     .main-section-container {
         display: flex;
         flex-direction: column;
         flex-grow: 1;
+        background-image: url('./background.svg'); 
```

Comment:
We should make sure that build-tool / linter can identify if the image exists.

Reply:
We can import this image and use as background layer in the `index.tsx`